test/app_ref.c

```diff
@@ -92,6 +93,19 @@ main(void)
 
     TESTING("library shutdown with reference count > 1");
 
+    /* Get the VFD to use */
+    env_h5_drvr = HDgetenv("HDF5_DRIVER");
+    if (env_h5_drvr == NULL)
+        env_h5_drvr = "nomatch";
+
```

Comment:
same here and in the tests in general, I would avoid using strings for const names, e.g. "multi" etc